feat: add first remote working chapter draft (#234)
Il-Libro-Open-Source/book: docs/it/remote-working.md

# Remote working: tra mito e realtà

A noi developers basta un portatile e una connessione internet per lavorare, _almeno è così che piace pensare a molti_.

Nella realtà dei fatti il lavoro da remoto è un argomento molto discusso e divisivo, con aziende che non lo permettono, aziende che lo sposano al 100% e altre che hanno trovato una soluzione di mezzo.

> Non sono solo le aziende però ad avere idee contrastanti, anche all'interno della community dei dev ci si divide tra chi non metterebbe mai più piede in un ufficio, chi non sa stare senza la pausa caffè alle macchinette coi colleghi e chi semplicemente, anche questa volta, sta nel mezzo.

Reviewer: ```suggestion
Non sono solo le aziende però ad avere idee contrastanti, anche all'interno della community dev ci si divide tra chi non metterebbe mai più piede in un ufficio, chi non sa stare senza la pausa caffè alle macchinette con colleghi e colleghe e chi semplicemente, anche questa volta, sta nel mezzo.
```

> A parte i freelance e tutti i dev che lavorano in maniera indipendente, le possibilità di lavoro da remoto all'interno delle aziende sono cambiate molto negli anni, con un vero e proprio sparti-acque tra pre e post pandemia da COVID-19.

Reviewer: ```suggestion
A parte freelance e dev che lavorano in maniera indipendente, le possibilità di lavoro da remoto all'interno delle aziende sono cambiate molto negli anni, con un vero e proprio sparti-acque tra _pre_ e _post_ pandemia da COVID-19.
```

## In principio fu il Telelavoro

In precedenza l'unica possibilità per convincere la proria azienda a poter lavorare da remoto era fare richiesta di telelavoro, ovvero allestendo un vero e proprio ufficio presso la propria abitazione.

Non era una condizione disponibile per chiunque ma spesso era legata a delle questioni di necessità proprie o di propri familiari, che richiedevano la permanenza a casa del dipendente.

E non si trattava nemmeno di un semplice lascia-passare: bisognava allestire letteralmente una postazione a casa che rispettasse gli stessi vincoli che si hanno in un ufficio, come disposizione della scrivania, altezza della sedia; insomma _non proprio una passeggiata_.

## Smart working: lavorare in maniera intelligente

Col tempo le aziende hanno iniziato a sperimentare una nuova modalità di lavoro, in cui si fornisce al dipendente la libertà di lavorare in posti differenti dall'ufficio e di gestire la propria attività lavorativa in autonomia: lo smart working appunto.

> Con la pandemia però, e la necessità di far lavorare le persone da casa, si è diffusa la convizione che _lavorare in smart working = lavorare da casa_, credenza tutta italiana visto che nel resto del mondo anglofono quest'ultima frase si traduce semplicemente in _work from home_.

Author: Hai ragione, però in questo contesto volevo usare proprio il termine specifico work from home perché è quello che è stato utilizzato nello shift da ufficio a casa durante la pandemia.

> Lo smart working quindi è un concetto più esteso, che parte dal poter lavorare lontano dal proprio ufficio, ma definisce anche come gestirsi l'attività quotidiana di lavoro, non ragionando più sulle 8 ore giornaliere, ma sugli obiettivi da raggiungere con il proprio lavoro.

Reviewer: ```suggestion
Lo smart working quindi è un concetto più esteso, che parte dal poter lavorare lontano dal proprio ufficio, ma definisce anche come gestirsi l'attività quotidiana di lavoro, non ragionando più sulle 8 ore giornaliere, ma sugli obiettivi da raggiungere nel proprio lavoro.
```

> Il vantaggio di avere questo tipo di autonomia è che si riesce ad avere una work-life balance migliore, per esempio non dovendo prendere più permessi per semplici commissioni, e stimola maggiormente il raggiungimento degli obiettivi, che sostituiscono il tempo come unità di misura del proprio lavoro.

Reviewer: ```suggestion
Il vantaggio di avere questo tipo di autonomia, se correttamente sfruttata, è che si riesce ad avere una _work-life balance_ migliore, per esempio non dovendo prendere più permessi per semplici commissioni, e stimola maggiormente il raggiungimento degli obiettivi, che sostituiscono il tempo come unità di misura del proprio lavoro.
```

Con questa modalità inoltre non si elimina il lavoro in presenza, ma lo si alterna con quello da remoto, spesso con una ratio ufficio/remoto definita insieme all'azienda, per esempio 2 giorni in ufficio e 3 da remoto a settimana, anche se col tempo, e dimostrando di essere efficienti in qualunque modalità, questa rigidità spesso si allenta.

Possiamo quindi vedere nello smart working la diretta evoluzione del classico lavoro d'ufficio con orario 9-18, con benifici sulla qualità della vita dei dipendenti e ottimizzazione dei costi da parte delle aziende.

## Full remote o morte

Spesso però l'alternanza ufficio/casa non è abbastanza, sia per il developer che per l'azienda, ed ecco che il sogno diventa realtà: si lavora esclusivamente da remoto.

> In questo caso si tratta di aziende strutturate per avere il proprio team completamente a distanza, che decidono così di abbattere completamente o quasi i costi di gestione di un ufficio ed avere il personale dislocato in luoghi diversi, spesso anche in nazioni o addirittura continenti diversi.

Reviewer: ```suggestion
In questo caso si tratta di aziende strutturate per avere il proprio team completamente a distanza, che decidono così di abbattere completamente - o quasi - i costi di gestione di un ufficio ed avere il personale dislocato in luoghi diversi, spesso anche in nazioni o addirittura continenti diversi.
```

> Questo sblocca molte possibilità a noi developers:

Reviewer: ```suggestion
Questo sblocca molte possibilità a noi developer:
```

- non doverci trasferire se la nostra area non offre possibilità di lavoro;
- lavorare per aziende estere, cosa che spesso risulta economicamente vantaggiosa;
- spostarci e lavorare viaggiando, cosa di cui parleremo più avanti nel capitolo.

> I vantaggi principali per le aziende invece sono, come già detto, l'abbattimento dei costi fissi (no ufficio, no bollette) e la possibilità di assumere developers che ricoprano con più precisione i profili ricercati, non dovendosi limitare a cercare tra persone della stessa area geografica o disposte a trasferirsi.

Reviewer: ```suggestion
I vantaggi principali per le aziende invece sono, come già detto, l'abbattimento dei costi fissi (no ufficio, no bollette) e la possibilità di assumere developer che ricoprano con più precisione i profili ricercati, non dovendosi limitare a cercare tra persone della stessa area geografica o disposte a trasferirsi.
```

> Il lavoro in un team full remote viene gestito adottando diverse tecniche della metodologia Agile, come gli stand up mattutini, meeting schedulati giornalmente in cui ogni dev racconta il lavoro fatto il giorno precedente e cosa andrà a svolgere nella giornata, in modo da condividere in tutto il team lo stato corrente di lavoro sui progetti in corso.

Reviewer: ```suggestion
Il lavoro in un team full remote viene gestito adottando diverse tecniche della metodologia Agile, come gli stand up mattutini, meeting schedulati giornalmente in cui ogni dev racconta il lavoro fatto il giorno precedente e cosa andrà a svolgere nella giornata, in modo da condividere con tutto il team lo stato corrente di lavoro sui progetti in corso.
```

## La leggenda dei nomadi digitali

> Se la maggior parte di chi lavora da remoto si limita a organizzarsi una scrivania fantascientifica a casa o al massimo va in qualche coworking in città ogni tanto, c'è però un sottogruppo di persone che sfrutta questa libertà per viaggiare per il mondo: i famosi _nomadi digitali_.

Reviewer: ```suggestion
Se la maggior parte di chi lavora da remoto si limita a organizzarsi una scrivania fantascientifica a casa o al massimo va in qualche co-working in città ogni tanto, c'è un sottogruppo di persone che sfrutta questa libertà per viaggiare per il mondo: i famosi _nomadi digitali_.
```

> Se fino a qualche anno fa lavorare viaggiando era prerogativa di lavoratori freelance o imprenditori più o meno di successo, ora che si può lavorare più facilmente in full remote ecco che sempre più persone decidono di spostarsi qualche mese in altri paesi per lavorare in posti diversi.

Reviewer: ```suggestion
Se fino a qualche anno fa lavorare viaggiando era prerogativa di lavoratori freelance o imprenditori più o meno di successo, ora che si può lavorare pressoché ovunque ecco che sempre più persone decidono di spostarsi qualche mese in altri paesi per lavorare.
```
"in posti diversi" mi sembra ridondante avendo già indicato "in altri paesi"

I vantaggi di lavorare viaggiando sono molteplici:

- esplorare nuovi paesi senza dover prendere ferie;
- vivere per lunghi periodi in paesi con costo della vita più basso;

> - cercare un clima migliore durante l'inverno;

Reviewer: ```suggestion
- cercare un clima più adatto alle proprie abitudini - periodi caldi d'inverno, freschi d'estate, per dirne una!
```

> - fare nuove esperienze appena fuori dall'orario di lavoro.

Reviewer: ```suggestion
- fare nuove esperienze fuori dall'orario di lavoro.
```

> Ovviamente, si è creata una visione idealistica del nomade digitale, e non sono poche le difficoltà:

Reviewer: ```suggestion
Ovviamente, questo può creare una visione idealistica del nomade digitale, ma non sono poche le difficoltà:
```

- necessità di adattarsi ad ogni nuovo spostamento;
- avere a che fare con fuso orari anche molto diversi da quello della propria azienda;
- problemi di socialità viaggiando in solitaria;

> - problemi tecnici, come connessioni scadenti, che causano poi problemi nel lavorare.

Reviewer: ```suggestion
- problemi tecnici, come connessioni scadenti, che possono causare problemi nel lavorare.
```

Tutti questi fattori sono assolutamente da tenere in considerazione se si valuta di partire per lavorare da altri paesi, e il consiglio per minimizzare l'impatto del viaggio è concedersi dei giorni senza lavorare appena arrivati in un nuovo paese, per poter verificare le modalità di lavoro che ci saranno, scegliere paesi che non richiedano di lavorare in orari assurdi, prediligere località _nomad friendly_ per conoscere altri viaggiatori che lavorano e avere più di una soluzione per quanto riguarda la propria postazione di lavoro nomade.

Esistono molte community online di altri nomadi digitali e siti che permettono di valutare le mete preferibili per iniziare, uno su tutti [Nomad list](https://nomadlist.com/), ma in assoluto viaggiare lavorando è una pratica sempre più diffusa che può davvero cambiare scenario alle giornate di lavoro.

## Programmare da remoto funziona davvero?

> Se il lavoro da remoto porta molti vantaggi sulla gestione della proprio vita privata, ci sono però aspetti tecnici che infiammano le discussioni tra noi developers, con molte criticità che possono presentarsi con questa modalità di lavoro.

Reviewer: ```suggestion
Se il lavoro da remoto porta molti vantaggi sulla gestione della proprio vita privata, ci sono però aspetti tecnici che infiammano le discussioni tra noi developer, con molte criticità che possono presentarsi con questa modalità di lavoro.
```

> La più evidente riguarda la formazione dei developers junior, che entrando in un'azienda completamente da remoto non hanno la possibilità di lavorare letteralmente _fianco fianco_ ai developers senior e quindi non riescono ad apprendere velocemente dall'esperienza e si ritrovano isolati in una fase molto importante della loro formazione.

Reviewer: ```suggestion
La più evidente riguarda la formazione delle figure junior, che entrando in un'azienda completamente da remoto non hanno la possibilità di lavorare letteralmente _fianco a fianco_ a figure senior e perdono la possibilità di apprendere velocemente dall'esperienza, ritrovandosi isolati in una fase molto importante della loro formazione.
```

> In realtà a questo si può in parte sopperire con uno sforzo maggiore dell'azienda e del resto del team a implementare un processo di on-boarding strutturato, sessioni dedicate esclusivamente alla formazione e permettere ai dev con meno seniority di poter facilmente avere supporto in caso di problematiche, tecniche e non.

Reviewer: ```suggestion
In realtà a questo si può in parte sopperire con uno sforzo maggiore dell'azienda e del resto del team a implementare un processo di on-boarding strutturato, sessioni dedicate esclusivamente alla formazione e permettere a dev con meno seniority di poter facilmente avere supporto in caso di problematiche, tecniche e non.
```

Un'altra criticità riguarda la difficoltà di coordinare il lavoro, con meeting fatti in videochiamata poco produttivi rispetto a sessioni di brain storming di persona e più in generale poca coesione all'interno del team rispetto ad un gruppo di developer che lavorano nella stessa stanza.

Questa è sicuramente una possibilità concreta ed è innegabile che l'essere fisicamente con altre persone non sia la stessa cosa di una chiamata su Zoom, è anche per questo che molte aziende decidono comunque di avere una sede o degli uffici a disposizione dei dipendenti, che sono liberi di scegliere se e quando andare.

> Infine spesso si evidenzia un aumento del turn over all'interno delle aziende, ovvero è più facile che un developer che lavora da remoto si licenzi perché non ha creato un reale legame con i colleghi e quel senso di appartenenza che fa si che si crei un clima aziendale accogliente.

Reviewer: ```suggestion
Infine spesso si evidenzia un aumento del turn over all'interno delle aziende, ovvero è più facile che chi lavora da remoto si licenzi perché non è riuscito a creare un reale legame con i colleghi né quel senso di appartenenza aziendale che contraddistingue solitamente chi lavora in ufficio.
```

> Questa criticità dovrebbe essere supportata da dati più concreti, ma è dimostrabile come il lavoro in presenza non sia sinonimo di clima aziendale positivo e che molte aziende full remote riescano a creare una sinergia tra le persone grazie alla condivisione di valori e all'organizzazione di eventi in persona, per fare il famoso team building, dimostrando un reale interesse a creare un gruppo coeso e soddisfatto del proprio lavoro.

Reviewer: Questa frase la riscriverei.. Capisco il senso, ma comincia come se fosse un proseguio della precedente mentre è letteralmente l'opposto.. Non saprei cosa consigliare, magari ti viene in mente un altro modo di esprimerlo!
Altra faccia della medaglia? D'altro canto? Non so..

Author: CI sta, ho provato a sottolineare l'anternanza

> È evidente che non ci sia una risposta definitiva alla domanda iniziale, ma è molto probabile che come spesso accade la risposta stia nel mezzo: lavorare da remoto funziona se si vuole farlo funzionare, accentandone i limiti e lasciando la libertà alle persone di poter decidere come organizzare al meglio il proprio lavoro e la propria vita.

Reviewer: ```suggestion
È evidente che non ci sia una risposta definitiva alla domanda che ci siamo posti qualche riga fa, ma è molto probabile che come spesso accade la risposta stia nel mezzo: lavorare da remoto funziona se si vuole farlo funzionare, accentandone i limiti e lasciando la libertà alle persone di poter decidere come organizzare al meglio il proprio lavoro e la propria vita.
```

Ho cercato all'inizio la domanda per poi accorgermi che era dopo la riga 70 ahahah

# Remote working: tra mito e realtà

A noi developers basta un portatile e una connessione internet per lavorare, _almeno è così che piace pensare a molti_.

Nella realtà dei fatti il lavoro da remoto è un argomento molto discusso e divisivo, con aziende che non lo permettono, aziende che lo sposano al 100% e altre che hanno trovato una soluzione di mezzo.

Non sono solo le aziende però ad avere idee contrastanti, anche all'interno della community dev ci si divide tra chi non metterebbe mai più piede in un ufficio, chi non sa stare senza la pausa caffè alle macchinette con colleghi e colleghe e chi semplicemente, anche questa volta, sta nel mezzo.

> In questo capitolo attraversiamo il mondo del lavoro da remoto, cercando di capirne vantaggi e svantaggi, con la premessa che chi ha scritto inizialmente il capitolo è un nomade digitale e quindi porta qui anche un po' della propria esperienza personale.

Author: @Cadienvan ho aggiunto frase nella intro per riportare il tuo suggerimento sull'exp dello scrittore, non volevo fare riferimenti ma giusto dare un minimo di contesto.

> A parte freelance e dev che lavorano in maniera indipendente, le possibilità di lavoro da remoto all'interno delle aziende sono cambiate molto negli anni, con un vero e proprio sparti-acque tra _pre_ e _post_ pandemia da COVID-19.

Reviewer 2: Sennò sembra che lavori da remoto nell'ufficio :D

```suggestion
A parte freelance e dev che lavorano in maniera indipendente, le possibilità di lavoro da remoto come dipendenti sono cambiate molto negli anni, con un vero e proprio sparti-acque tra _pre_ e _post_ pandemia da COVID-19.
```

## In principio fu il Telelavoro

> In precedenza l'unica possibilità per convincere la proria azienda a poter lavorare da remoto era fare richiesta di telelavoro, ovvero allestendo un vero e proprio ufficio presso la propria abitazione.

Reviewer 2: Typo

```suggestion
In precedenza l'unica possibilità per convincere la propria azienda a poter lavorare da remoto era fare richiesta di telelavoro, ovvero allestendo un vero e proprio ufficio presso la propria abitazione.
```

Non era una condizione disponibile per chiunque ma spesso era legata a delle questioni di necessità proprie o di propri familiari, che richiedevano la permanenza a casa del dipendente.

E non si trattava nemmeno di un semplice lascia-passare: bisognava allestire letteralmente una postazione a casa che rispettasse gli stessi vincoli che si hanno in un ufficio, come disposizione della scrivania, altezza della sedia; insomma _non proprio una passeggiata_.

## Smart working: lavorare in maniera intelligente

Col tempo le aziende hanno iniziato a sperimentare una nuova modalità di lavoro, in cui si fornisce al dipendente la libertà di lavorare in posti differenti dall'ufficio e di gestire la propria attività lavorativa in autonomia: lo smart working appunto.

Con la pandemia però, e la necessità di far lavorare le persone da casa, si è diffusa la convizione che _lavorare in smart working = lavorare da casa_, credenza tutta italiana visto che nel resto del mondo anglofono quest'ultima frase si traduce semplicemente in _work from home_.

> Lo smart working quindi è un concetto più esteso, che parte dal poter lavorare lontano dal proprio ufficio, ma definisce anche come gestirsi l'attività quotidiana di lavoro, non ragionando più sulle 8 ore giornaliere, ma sugli obiettivi da raggiungere nel proprio lavoro.

Reviewer 2: Nit: ci sono CCNL dove il monte ore è vario

```suggestion
Lo smart working quindi è un concetto più esteso, che parte dal poter lavorare lontano dal proprio ufficio, ma definisce anche come gestirsi l'attività quotidiana di lavoro, non ragionando più sulle classiche ore giornaliere previste dal proprio CCNL (Contratto Collettivo Nazionale del Lavoro), ma sugli obiettivi da raggiungere nel proprio lavoro.
```

Il vantaggio di avere questo tipo di autonomia, se correttamente sfruttata, è quello di riuscire ad avere una _work-life balance_ migliore, per esempio non dovendo prendere più permessi per semplici commissioni, stimolando maggiormente il raggiungimento degli obiettivi, che sostituiscono il tempo come unità di misura del proprio operato.

> Con questa modalità inoltre non si elimina il lavoro in presenza, ma lo si alterna con quello da remoto, spesso con una ratio ufficio/remoto definita insieme all'azienda, per esempio 2 giorni in ufficio e 3 da remoto a settimana, anche se col tempo, e dimostrando di essere efficienti in qualunque modalità, questa rigidità spesso si allenta.

Reviewer 2: ```suggestion
Con questa modalità inoltre non si elimina il lavoro in presenza, ma lo si alterna con quello da remoto, spesso con una ratio ufficio/remoto definita insieme all'azienda, per esempio 2 giorni in ufficio e 3 da remoto a settimana, anche se col tempo, e dimostrando di essere efficienti in qualunque modalità, questa rigidità potrebbe allentarsi.
```

Possiamo quindi vedere nello smart working la diretta evoluzione del classico lavoro d'ufficio con orario 9-18, con benifici sulla qualità della vita dei dipendenti e ottimizzazione dei costi da parte delle aziende.

## Full remote o morte

> Spesso però l'alternanza ufficio/casa non è abbastanza, sia per il developer che per l'azienda, ed ecco che il sogno diventa realtà: si lavora esclusivamente da remoto.

Reviewer 3: ```suggestion
Spesso però l'alternanza ufficio/casa non è abbastanza, sia per l'individuo che per l'azienda, ed ecco che il sogno diventa realtà: si lavora esclusivamente da remoto.
```

Author: Grazie ci tenevo, ho provato a farci attenzione il più possibile. ☺️

> In questo caso si tratta di aziende strutturate per avere il proprio team completamente a distanza, che decidono così di abbattere completamente - o quasi - i costi di gestione di un ufficio ed avere il personale dislocato in luoghi diversi, spesso anche in nazioni o addirittura continenti diversi.

Reviewer 2: Non mi suonava tanto bene

```suggestion
In questo caso parliamo di aziende strutturate per avere il proprio team completamente a distanza, che decidono così di abbattere completamente - o quasi - i costi di gestione di un ufficio ed avere il personale dislocato in luoghi diversi, spesso anche in nazioni o addirittura continenti diversi.
```

Questo sblocca molte possibilità per noi developer:

> - non doverci trasferire se la nostra area non offre possibilità di lavoro;
> - lavorare per aziende estere, cosa che spesso risulta economicamente vantaggiosa;
> - spostarci e lavorare viaggiando, cosa di cui parleremo più avanti nel capitolo.

Reviewer 2: Così sfruttiamo markdown

```suggestion
* non doverci trasferire se la nostra area non offre possibilità di lavoro;
* lavorare per aziende estere, cosa che spesso risulta economicamente vantaggiosa;
* spostarci e lavorare viaggiando, cosa di cui parleremo più avanti nel capitolo.
```

I vantaggi principali per le aziende invece sono, come già detto, l'abbattimento dei costi fissi (no ufficio, no bollette) e la possibilità di assumere developer che ricoprano con più precisione i profili ricercati, non dovendosi limitare a cercare tra persone della stessa area geografica o disposte a trasferirsi.

Il lavoro in un team full remote viene gestito adottando diverse tecniche della metodologia Agile, come gli stand up mattutini, meeting schedulati giornalmente in cui ogni dev racconta il lavoro fatto il giorno precedente e cosa andrà a svolgere nella giornata, in modo da condividere con tutto il team lo stato corrente del proprio lavoro sui progetti in corso.

## La leggenda dei nomadi digitali

Se la maggior parte di chi lavora da remoto si limita a organizzarsi una scrivania fantascientifica a casa o al massimo va in qualche co-working in città ogni tanto, c'è un sottogruppo di persone che sfrutta questa libertà per viaggiare per il mondo: i famosi _nomadi digitali_.

> Se fino a qualche anno fa lavorare viaggiando era prerogativa di lavoratori freelance o imprenditori più o meno di successo, ora che si può lavorare pressoché ovunque ecco che sempre più persone decidono di spostarsi qualche mese in altri paesi per lavorare.

Reviewer 2: ```suggestion
Se fino a qualche anno fa lavorare viaggiando era prerogativa di lavoratori freelance o imprenditori più o meno di successo, ora che si può lavorare pressoché ovunque ecco che sempre più persone decidono di spostarsi per lavorare in altri paesi per un periodo di tempo non precisamente definito.
```

I vantaggi di lavorare viaggiando sono molteplici:

> - esplorare nuovi paesi senza dover prendere ferie;
> - vivere per lunghi periodi in paesi con costo della vita più basso;
> - cercare un clima più adatto alle proprie abitudini - periodi caldi d'inverno, freschi d'estate, ad esempio;
> - fare nuove esperienze fuori dall'orario di lavoro.

Reviewer 2: ```suggestion
* esplorare nuovi paesi senza dover prendere ferie;
* vivere per lunghi periodi in paesi con costo della vita più basso;
* cercare un clima più adatto alle proprie abitudini - periodi caldi d'inverno, freschi d'estate, ad esempio;
* fare nuove esperienze fuori dall'orario di lavoro.
```

Ovviamente, questo può creare una visione idealistica del nomade digitale, ma non sono poche le difficoltà:

> - necessità di adattarsi ad ogni nuovo spostamento;
> - avere a che fare con fuso orari anche molto diversi da quello della propria azienda;
> - problemi di socialità se si viaggia in solitaria;
> - problemi tecnici, come connessioni scadenti, che possono causare problemi nel lavorare.

Reviewer 2: ```suggestion
* necessità di adattarsi ad ogni nuovo spostamento;
* avere a che fare con fusi orari anche molto diversi da quello della propria azienda;
* problemi di socialità se si viaggia in solitaria;
* problemi tecnici, come connessioni scadenti, che possono causare problemi di comunicazione.
```

Tutti questi fattori sono assolutamente da tenere in considerazione se si valuta di partire per lavorare da altri paesi, e il consiglio per minimizzare l'impatto del viaggio è concedersi dei giorni senza lavorare appena arrivati in un nuovo paese, per poter verificare le modalità di lavoro che ci saranno, scegliere paesi che non richiedano di lavorare in orari assurdi, prediligere località _nomad friendly_ per conoscere altri viaggiatori che lavorano e avere più di una soluzione per quanto riguarda la propria postazione di lavoro nomade.

Esistono molte community online di altri nomadi digitali e siti che permettono di valutare le mete preferibili per iniziare, uno su tutti [Nomad list](https://nomadlist.com/), ma in assoluto viaggiare lavorando è una pratica sempre più diffusa che può davvero cambiare scenario alle giornate di lavoro.

## Programmare da remoto funziona davvero?

> Se il lavoro da remoto porta molti vantaggi sulla gestione della proprio vita privata, ci sono però aspetti tecnici che infiammano le discussioni tra noi developer, con molte criticità che possono presentarsi con questa modalità di lavoro.

Reviewer 2: ```suggestion
Se il lavoro da remoto porta molti vantaggi sulla gestione della propria vita privata, ci sono però aspetti tecnici che infiammano le discussioni tra noi developer, con molte criticità che possono presentarsi con questa modalità di lavoro.
```

La più evidente riguarda la formazione delle figure junior, che entrando in un'azienda completamente da remoto non hanno la possibilità di lavorare letteralmente _fianco a fianco_ a figure senior e perdono la possibilità di apprendere velocemente dall'esperienza, ritrovandosi isolati in una fase molto importante della loro formazione.

> In realtà a questo si può in parte sopperire con uno sforzo maggiore dell'azienda e del resto del team a implementare un processo di on-boarding strutturato, sessioni dedicate esclusivamente alla formazione e permettere a dev con meno seniority di poter facilmente avere supporto in caso di problematiche, tecniche e non.

Reviewer 2: ```suggestion
In realtà a questo si può in parte sopperire con uno sforzo maggiore dell'azienda e del resto del team impostando un processo di on-boarding strutturato, sessioni dedicate esclusivamente alla formazione e permettere a dev con meno seniority di poter facilmente avere supporto in caso di problematiche, tecniche e non.
```

> Un'altra criticità riguarda la difficoltà di coordinare il lavoro, con meeting fatti in videochiamata poco produttivi rispetto a sessioni di brain storming di persona e più in generale poca coesione all'interno del team rispetto ad un gruppo di developer che lavorano nella stessa stanza.

Reviewer 2: ```suggestion
Un'altra criticità riguarda la difficoltà di coordinare il lavoro, con meeting fatti in videochiamata poco produttivi rispetto a sessioni di brain storming di persona, e più in generale poca coesione all'interno del team rispetto ad un gruppo di developer che lavorano nella stessa stanza.
```

> Un'altra criticità riguarda la difficoltà di coordinare il lavoro, con meeting fatti in videochiamata poco produttivi rispetto a sessioni di brain storming di persona e più in generale poca coesione all'interno del team rispetto ad un gruppo di developer che lavorano nella stessa stanza.

Reviewer 2: Pensi sia il posto e il momento giusto per parlare anche di questioni strettamente previdenziali/assicurative? Ad esempio alcune aziende non permettono ai propri dipendenti di lavorare oltre una certa quantità di giorni all'estero

Author: Mancava questo punto, grazie. L'ho inserito in quella parte di capitolo, dimmi cosa ne pensi e correggi pure se necessario :). Ho dovuto anche rimuovere le correzioni alle liste che mi avevi fatto perché mi davano errore nel format:check (* al posto di -)

Questa è sicuramente una possibilità concreta ed è innegabile che l'essere fisicamente con altre persone non sia la stessa cosa di una chiamata su Zoom, è anche per questo che molte aziende decidono comunque di avere una sede o degli uffici a disposizione dei dipendenti, che sono liberi di scegliere se e quando andare.

> Infine spesso si evidenzia un aumento del turn over all'interno delle aziende, ovvero è più facile che chi lavora da remoto si licenzi perché non è riuscito a creare un reale legame con i colleghi né quel senso di appartenenza aziendale che contraddistingue solitamente chi lavora in ufficio.

Reviewer 2: ```suggestion
Infine spesso si evidenzia un aumento del turnover all'interno delle aziende, ovvero è più facile che chi lavora da remoto si licenzi perché non è riuscito a creare un reale legame con i colleghi né quel senso di appartenenza aziendale che contraddistingue solitamente chi lavora in ufficio.
```

Tuttavia è dimostrabile come il lavoro in presenza non sia sinonimo di clima aziendale positivo e che molte aziende full remote riescano a creare una sinergia tra le persone grazie alla condivisione di valori e all'organizzazione di eventi in persona, per fare il famoso team building, dimostrando un reale interesse a creare un gruppo coeso e soddisfatto del proprio lavoro.

> È evidente che non ci sia una risposta definitiva alla domanda che ci siamo posti qualche riga fa, ma è molto probabile che come spesso accade la risposta stia nel mezzo: lavorare da remoto funziona se si vuole farlo funzionare, accentandone i limiti e lasciando la libertà alle persone di poter decidere come organizzare al meglio il proprio lavoro e la propria vita.

Reviewer 2: ```suggestion
È evidente che non ci sia una risposta definitiva alla domanda che ci siamo posti qualche riga fa, ma è molto probabile che come spesso accade la risposta stia nel mezzo: lavorare da remoto funziona se si vuole farlo funzionare, accettandone i limiti e lasciando la libertà alle persone di poter decidere come organizzare al meglio il proprio lavoro e la propria vita.
```
